library/std/src/sys/pal/unix/net.rs

```diff
@@ -74,7 +74,6 @@ impl Socket {
                 if #[cfg(any(
                     target_os = "android",
                     target_os = "dragonfly",
-                    target_os = "freebsd",
```

Comment:
This makes an extra syscall to call set_cloexec necessary, right?

Reply:
yes